### ISSUE TITLE
Audio Doesn't work with Safari 11(Mac OS)

### ISSUE THREAD

**Issue by divya-kanak** [twilio/video-quickstart-js#103]
I have migrated the code from version 1+ to version 2+ using [this](https://www.twilio.com/docs/verify/migrating-1x-2x). 

As I have search out for the issue is **Audio Doesn't work with Safari 11(Mac OS)** And I get the idea about the codec.

So, I have set the below codec for audio and video 

`preferredAudioCodecs: ['isac', 'OPUS']`

`if (isChrome() || isFirefox()) {
    connectOptions.preferredVideoCodecs = ['VP9', 'H264'];
  } else {
    connectOptions.preferredVideoCodecs = ['H264', 'VP9'];
  }`

Currently, this is running well with all browsers. But when I trying to test between chrome to safari. At times Audio is not working at both sides.

**Comment by makarandp0** [comment]
Hello @divya-kanak, Thank you for writing about this issue. Were you able to get quick-start sample working on Safari? Did you see audio issues in quick-start as well or just your application ? Does the audio issue on safari persist if you omit `preferredVideoCodecs` ?

BTW your migrating from 1x to 1x link pointed to:  https://www.twilio.com/docs/verify/migrating-1x-2x but I assume you meant: https://www.twilio.com/docs/video/migrating-1x-2x

**Comment by divya-kanak** [comment]
Yes, I have used this link https://www.twilio.com/docs/video/migrating-1x-2x
This was coming into my application and I have not tested your quick-start semple still now.
Only the Audio issue is there.

After checking this issue as long as and Twilio support. I have changed the codec for audio as below : 

`preferredAudioCodecs: ['isac', 'opus', 'PCMA', 'PCMU']`

Now, audio is working fine with safari versions 11 and 12.
